pyprland/common.py

```diff
@@ -181,6 +181,10 @@ def get_logger(name="pypr", level=None) -> logging.Logger:
     return logger
 
 
+class VersionInfo(tuple[int, int, int]):
+    pass
+
+
```

Comment:
This code is fine, but maybe you are interested in dataclasses here https://docs.python.org/3/library/dataclasses.html

Reply:
I will update this